plugins/ui/nvim-highlight-colors.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (helpers.defaultNullOpts)
+    mkBool
+    mkEnum
+    mkListOf
+    mkStr
+    mkStr'
+    ;
+in
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "nvim-highlight-colors";
+  defaultPackage = pkgs.vimPlugins.nvim-highlight-colors;
```

Watch out for #2139, if it is merged first you'll have to do:

```suggestion
  package = "nvim-highlight-colors";
```